ImgSmall.java

```diff
@@ -0,0 +1,48 @@
+import javax.imageio.ImageIO;
+import java.awt.Image;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+
+public class ImgSmall {
+  public static void main(String[] args) throws IOException {
+      resize(args);
+  }
+  public static void resize(String[] args) throws IOException{
+    //get url object
+    URL url = null;
+    try{
+      url = new URL(args[0]);
+    } catch (Exception MalformedURLException){
+      System.out.println("check the url");
+      System.exit(0);
+    }
+    double resizeBy = (100-Integer.parseInt(args[1]))/100.0;
+
+    //get image from url
+    BufferedImage img = ImageIO.read(url);
+    //resize image into Image object, SCALE_FAST is a random algorithm
+    Image tmpImage = img.getScaledInstance((int)(img.getWidth() * resizeBy),
+        (int)(img.getHeight() * resizeBy),
+        BufferedImage.SCALE_FAST);
+    //Create resized BufferedImage object -- this is currently empty
+    BufferedImage buffered = new BufferedImage((int)(img.getWidth() * resizeBy),
```

Comment:
Instead of "buffered" the name of the variable should be bufferedImage, as later in the code "buffered" is meaningless and the reader has to scroll up to see what does the buffer contain (as in this case this is image bytes)